### PR TITLE
[SUSE] add registered trademark tag to the product name

### PR DESCRIPTION
**Pull request by MaximilianMeister**

### CODE REVIEW

File: crowbar_framework/app/controllers/ucs_controller.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #
 
 # Things that still should be done:
-# - apply SUSE OpenStack Cloud admin CSS to views
+# - apply SUSE© OpenStack Cloud admin CSS to views
```
Comment (vuntz) [inline crowbar_framework/app/controllers/ucs_controller.rb]: hrm, do we really need it in comments? :-)

File: crowbar_framework/config/branding.yml [hunk 2]
```diff
@@ -16,17 +16,17 @@
 
 # The main page title that gets displayed on all pages on the top bar and within
 # the meta information.
-page_title: 'SUSE OpenStack Cloud Crowbar'
+page_title: 'SUSE<span class="copyright-tag">©</span> OpenStack Cloud'
```
Comment (vuntz) [inline crowbar_framework/config/branding.yml]: Isn't the page title also the title of the browser window (through the title tag in the HTML head)? Does it work to have HTML tags there? (it seems not, in my testing)

Comment (MaximilianMeister) [inline crowbar_framework/config/branding.yml]: we could use another key for the use in the header. sth like `product_name:`

File: crowbar_framework/app/controllers/ucs_controller.rb (hunk 1 above)
Comment (MaximilianMeister) [inline crowbar_framework/app/controllers/ucs_controller.rb]: not sure. you tell me :) I wanted to be consistent ;)

Comment (vuntz) [inline crowbar_framework/app/controllers/ucs_controller.rb]: Well, if it's up to me, I wouldn't put that in the comments :-)

File: crowbar_framework/public/404.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <title>
-      SUSE OpenStack Cloud Crowbar: 404 Error
+      SUSE<span class="copyright-tag">©</span> OpenStack Cloud Crowbar: 404 Error
```
Comment (vuntz) [inline crowbar_framework/public/404.html]: You can already remove the span tag from the static page while we decide how to solve the issue for the non-static pages ;-)

Comment (MaximilianMeister) [inline crowbar_framework/public/404.html]: sorry I haven't checked. too bad that you cannot style the title like that :)

@vuntz but shall I remove the tag completely from the static pages?

File: crowbar_framework/config/branding.yml (hunk 2 above)
Comment (MaximilianMeister) [inline crowbar_framework/config/branding.yml]: @vuntz what do you think about adding a key here? other solution would be to patch the `en.yml`
maybe you have a better idea

Comment (vuntz) [inline crowbar_framework/config/branding.yml]: > @vuntz what do you think about adding a key here?

I think it's fine to add a key, but it should be done upstream first.

File: crowbar_framework/public/404.html (hunk 3 above)
Comment (vuntz) [inline crowbar_framework/public/404.html]: > @vuntz but shall I remove the tag completely from the static pages?

What do you mean? I would just remove the <span> but keep the copyright symbol in the metadata title.

**Comment by tboerger** [comment]
I would just skip the (c) within titles and on the shell like within crowbar_register. And for the rest you should use &copy;

File: crowbar_framework/public/404.html (hunk 3 above)
Comment (MaximilianMeister) [inline crowbar_framework/public/404.html]: >  I would just remove the but keep the copyright symbol in the metadata title.

now i have to ask what you mean. the sentence is not complete... ;-)

Comment (vuntz) [inline crowbar_framework/public/404.html]: Heh, it's github that is removing my html tag; I was saying remove the span tag.

Comment (tboerger) [inline crowbar_framework/public/404.html]: In general I don't think that a (c) tag makes sense? I know that only for (r) to have it as a part of a company name, but (c)? Where do this task come from?

Comment (MaximilianMeister) [inline crowbar_framework/public/404.html]: > Where do this task come from?

well we had the copyright-tag in the @kwwii mockup from the installer. I just took it from there and thought it is somehow important.

**Comment by MaximilianMeister** [comment]
not sure what to do here next. do we still need it? as I said I just took it from @kwwii mockups, but I am still unsure if we really need it. @vuntz what do you say?

File: crowbar_framework/public/404.html (hunk 3 above)
Comment (tboerger) [inline crowbar_framework/public/404.html]: Even on the website I can not see something like that :)

**Comment by vuntz** [comment]
@MaximilianMeister put it in

**Comment by MaximilianMeister** [comment]
@vuntz okay I have left out the styling for now, hope that is fine. adding an extra key to the branding would be just to style the text size of the ©. do you think that is really necessary or can we leave it like that?

copyright-tag

**Comment by tboerger** [comment]
I think we should ask @kwwii first if the (c) really is intended and makes sense. IMHO that is wrong.

**Comment by MaximilianMeister** [comment]
> I think we should ask @kwwii first if the (c) really is intended and makes sense. IMHO that is wrong.

I will ping him to have a look here

**Comment by kwwii** [comment]
As discussed, the copyright should be replaced with a registered trademark.

**Comment by kwwii** [comment]
suse-openstack-cloud-logo

This is how it should look...

**Comment by MaximilianMeister** [comment]
any update here? can we merge this?

**Comment by tboerger** [comment]
:+1:

**Comment by dirkmueller** [comment]
:+1:

**Comment by dirkmueller** [comment]
argh, this forgot to update 403 page :(

**Pull request merged**